clion/src/191/main/kotlin/org/rust/clion/profiler/RsProfilerExecutor.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.clion.profiler
+
+import com.intellij.profiler.ProfilerExecutorBase
+
+class RsProfilerExecutor : ProfilerExecutorBase() {
```

Comment:
Looks like it's redundant and we can use `com.intellij.profiler.clion.ProfilerExecutor` instead
It also fixes additional profiler icon in toolbar